app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        super().learn_skill("JavaScript")
+        super().learn_skill("HTML")
+        super().learn_skill("CSS")
```

Comment:
Instead of using `super().learn_skill()` to add skills in the `FrontendDeveloper` constructor, you should use the `extend()` method or `+=` operator to add multiple skills at once to the `skills` list. This aligns with the checklist requirement to use `extend()` or `+=` for adding multiple values to a list .